source/kraken/realtime.cpp

```diff
@@ -57,8 +57,9 @@ static bool is_handleable(const transit_realtime::TripUpdate& trip_update){
     // Case 2: the trip is not cancelled, but modified (ex: the train's arrival is delayed or the train's
     // departure is brought forward), we check the size of stop_time_update because we can't find a proper
     // enum in gtfs-rt proto to express this idea
-    if (trip_update.trip().schedule_relationship() == transit_realtime::TripDescriptor_ScheduleRelationship_SCHEDULED
-            && trip_update.stop_time_update_size()) {
+    if ((trip_update.trip().schedule_relationship() == transit_realtime::TripDescriptor_ScheduleRelationship_SCHEDULED ||
```

Comment:
I think that checking the features flags (`is_realtime_add_enabled` and `is_realtime_add_trip_enabled`) should be done here.

Reply:
I think you have right, Wa can add the flags logic inside this function (is_handleable).
Flags logic
https://github.com/CanalTP/navitia/pull/2689/files/bc83c537ac0a8fd0e60e119d38651f48bc72fbd8#diff-a1c1532aa9f44bfd3b8a77e208515e23R544